node-zerox/src/utils.ts

```diff
@@ -104,3 +108,30 @@ export const convertPdfToImages = async ({
     console.error("Error during PDF conversion:", err);
   }
 };
+
+// Convert each page (from docx) to a png and save that image to tmp
```

Comment:
Let's update the comment since we are looking at all file types (not just docx here)